<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSimulation</class>
 <widget class="QDialog" name="CQDlgSimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>At simulation end</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqResetScene">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Reset scene to initial state</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqRemoveNewObjects">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Remove new objects</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Time steps</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="qqTimeStep">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>25</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation dt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqDynTimeStep">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>45</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qqDynamicsDtLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Dynamics dt</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="qqInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings are default</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Dynamics</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="qqGravityZ">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>65</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGravityY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>65</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Gravity vector</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAdjustEngine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>45</y>
      <width>176</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Engine properties</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqContactPoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Display contact points</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGravityX">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>65</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Dynamics enabled, engine</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqEngineCombo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>25</y>
      <width>176</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulation</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqFullscreen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Full screen at simulation start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqScriptExecutionPasses">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>25</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqPauseWhenTimeHigher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Pause when simulation time higher than</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqPauseOnScriptError">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Pause on script error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqPauseTime">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>65</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation passes per frame (ppf)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMultiplicationCoefficient">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>45</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqRealTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>331</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Real-time simulation, multiplication factor</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqTimeStep</tabstop>
  <tabstop>qqDynTimeStep</tabstop>
  <tabstop>qqScriptExecutionPasses</tabstop>
  <tabstop>qqRealTime</tabstop>
  <tabstop>qqMultiplicationCoefficient</tabstop>
  <tabstop>qqPauseWhenTimeHigher</tabstop>
  <tabstop>qqPauseTime</tabstop>
  <tabstop>qqPauseOnScriptError</tabstop>
  <tabstop>qqFullscreen</tabstop>
  <tabstop>qqEnabled</tabstop>
  <tabstop>qqEngineCombo</tabstop>
  <tabstop>qqContactPoints</tabstop>
  <tabstop>qqAdjustEngine</tabstop>
  <tabstop>qqGravityX</tabstop>
  <tabstop>qqGravityY</tabstop>
  <tabstop>qqGravityZ</tabstop>
  <tabstop>qqResetScene</tabstop>
  <tabstop>qqRemoveNewObjects</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
